docs/paper/admin/reference/system-properties.md

```diff
@@ -0,0 +1,162 @@
+---
+slug: /reference/system-properties
+---
+
+# Paper System Properties 
+
+These system properties can be set when you start your server allowing for the configuration of various settings.
```

Comment:
```suggestion
These system properties can be set when you start your server allowing for the configuration of various advanced settings.
```

Reply:
in general, maybe a bigger disclaimer about "danger ahead"?

Reply:
maybe time for a `:::danger` tag here :)